spec.html

```diff
@@ -5546,6 +5749,111 @@ <h1>CopyDataProperties ( _target_, _source_, _excludedItems_ )</h1>
         <p>The target passed in here is always a newly created object which is not directly accessible in case of an error being thrown.</p>
       </emu-note>
     </emu-clause>
+
+    <emu-clause id="sec-newprivatename" aoid="NewPrivateName">
+      <h1>NewPrivateName ( _description_ )</h1>
+      <p>The abstract operation NewPrivateName takes argument _description_ (a String). It performs the following steps when called:</p>
+      <emu-alg>
+        1. Assert: Type(_description_) is String.
+        1. Return a new unique Private Name whose [[Description]] value is _description_.
```

Comment:
```suggestion
        1. Return a new Private Name whose [[Description]] is _description_.
```

This also probably doesn't need to be an AO.

Reply:
Done in https://github.com/tc39/ecma262/pull/1668/commits/f429e6d7c9e9992fad633022fa775c31879d22c6.